proto/README.md

```diff
@@ -76,6 +76,23 @@ events for multiple resources in the same product (e.g. buckets and
 objects within Cloud Storage) can use common messages in their
 definitions.
 
+The product and version parts must be all lower-case to conform with proto
+package naming conventions. A product may consist of multiple
+segments (e.g. `cloud.storage`) but if a single segment consists of
+multple words, they should be simply lower-cased. Example:
+`firebase.remoteconfig` for the Firebase Remote Config product.
```

Comment:
nit:
"Example: `firebase.remoteconfig` for the Firebase Remote Config product." 
sounds a bit weird in the middle of this paragraph.

I'd imagine:
"For example, `firebase.remoteconfig` is the product name for Firebase Remote Config."

Reply:
Done.